packages/jsii/lib/project-info.ts

```diff
@@ -137,6 +137,18 @@ export async function loadProjectInfo(
 
   const transitiveDependencies = Object.values(transitiveAssemblies);
 
+  const metadata = mergeMetadata(
+    {
+      jsii: {
+        pacmak: {
+          // When `true`, `jsii-pacmak` will use the `Jsii$Default` implementation in code generation even for dependencies.
+          hasDefaultInterfaces: true,
+        },
+      },
+    },
+    pkg.jsii?.metadata,
```

Comment:
Sooo, you expect package.json to look like:

```js
{
  "jsii": {
    "metadata": {
      "jsii": {
         "pacmak": { ... }
      }
    }
  }
}
```

Feels like a bit of a stutter to repeat `jsii`. I can sorta see how it makes sense, but what is `metadata` expected to be used for?

Reply:
Metadata is used to carry arbitrary information into the assembly... For example if you have a custom documentation generator (*cough cough*) and need to provide it with specific configuration (say... a marketing-blessed name for cloud services?) so it renders the content you need... Then you can add whatever you please in there.

So the repeated `jsii` in there is mostly just because it's `jsii`'s own metadata namespace...

An alternative I have considered was to drop down and key up as `jsii-pacmak` instead (so it's one fewer level deep)... I don't know if this would look better or not to you? It didn't make much of a difference to me; and I felt the "top-level key" thing was perhaps a good thing to have in the long run?

Reply:
Yeah it's fine. Just looked weird at first.